### ISSUE TITLE
Setup GraphQL mandatory parameters

### ISSUE THREAD

**Issue by juanjoman** [boostercloud/booster#622]
This is related to https://github.com/boostercloud/booster/issues/621. 

1. I have this command:
```
@Command({
  authorize: 'all',
})
export class DoThing {
  public constructor(readonly id: UUID, readonly amount: number) {} // Notice that both params aren't optional

  public static async handle(command: QuotaConsumed, register: Register): Promise<void> {
    register.events(new ThingWasDone(command.id, command.amount))
  }
}
```
2. I send a mutation without the `amount` param (I forgot for example)
3. No errors are thrown from GraphQL, `amount` is set to `undefined` and it's reduced by the entity, which can cause additional issues depending on your application (in my case, on #621).

GraphQL should return an error saying that a mandatory param hasn't been set, and avoiding registering events that don't match the command constructor's parameter constraint.

cc @alvaroloes

**Comment by NickSeagull** [comment]
Agree, the GraphQL schema should be generated with `!` marking the fields as strict.
